2024-01-16 21:18:23,193 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-01-16 21:18:23,194 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Configure stats pid to 5264
2024-01-16 21:18:23,195 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Loading settings from /home/filipeckijan/.config/wandb/settings
2024-01-16 21:18:23,196 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Loading settings from /mnt/c/Users/dziew/Documents/studia/ium/ium-projekt/models/DBSCAN/wandb/settings
2024-01-16 21:18:23,197 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-16 21:18:23,198 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-16 21:18:23,199 INFO    MainThread:5264 [wandb_init.py:_log_setup():526] Logging user logs to /mnt/c/Users/dziew/Documents/studia/ium/ium-projekt/models/DBSCAN/wandb/run-20240116_211823-mgatar51/logs/debug.log
2024-01-16 21:18:23,200 INFO    MainThread:5264 [wandb_init.py:_log_setup():527] Logging internal logs to /mnt/c/Users/dziew/Documents/studia/ium/ium-projekt/models/DBSCAN/wandb/run-20240116_211823-mgatar51/logs/debug-internal.log
2024-01-16 21:18:23,201 INFO    MainThread:5264 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fecb6b25cc0>
2024-01-16 21:18:23,203 INFO    MainThread:5264 [wandb_init.py:init():566] calling init triggers
2024-01-16 21:18:23,203 INFO    MainThread:5264 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-01-16 21:18:23,204 INFO    MainThread:5264 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 9it338ja
2024-01-16 21:18:23,208 INFO    MainThread:5264 [wandb_run.py:_finish():1962] finishing run ium-team/ium-projekt/9it338ja
2024-01-16 21:18:23,209 INFO    MainThread:5264 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-16 21:18:23,211 INFO    MainThread:5264 [wandb_run.py:_restore():2185] restore
2024-01-16 21:18:23,212 INFO    MainThread:5264 [wandb_run.py:_restore():2191] restore done
2024-01-16 21:18:28,461 INFO    MainThread:5264 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-01-16 21:18:28,462 INFO    MainThread:5264 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-01-16 21:18:28,476 INFO    MainThread:5264 [wandb_run.py:_footer_sync_info():3816] logging synced files
2024-01-16 21:18:28,526 INFO    MainThread:5264 [wandb_init.py:init():616] starting backend
2024-01-16 21:18:28,527 INFO    MainThread:5264 [wandb_init.py:init():620] setting up manager
2024-01-16 21:18:28,541 INFO    MainThread:5264 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-01-16 21:18:28,542 INFO    MainThread:5264 [wandb_init.py:init():628] backend started and connected
2024-01-16 21:18:28,571 INFO    MainThread:5264 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-01-16 21:18:28,572 INFO    MainThread:5264 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-01-16 21:18:28,572 INFO    MainThread:5264 [wandb_init.py:init():720] updated telemetry
2024-01-16 21:18:28,689 INFO    MainThread:5264 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-01-16 21:18:30,081 INFO    MainThread:5264 [wandb_run.py:_on_init():2254] communicating current version
2024-01-16 21:18:30,321 INFO    MainThread:5264 [wandb_run.py:_on_init():2263] got version response 
2024-01-16 21:18:30,321 INFO    MainThread:5264 [wandb_init.py:init():804] starting run threads in backend
2024-01-16 21:18:30,497 INFO    MainThread:5264 [wandb_run.py:_console_start():2233] atexit reg
2024-01-16 21:18:30,498 INFO    MainThread:5264 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-01-16 21:18:30,499 INFO    MainThread:5264 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-01-16 21:18:30,500 INFO    MainThread:5264 [wandb_run.py:_redirect():2178] Redirects installed.
2024-01-16 21:18:30,502 INFO    MainThread:5264 [wandb_init.py:init():847] run started, returning control to user process
2024-01-16 21:18:30,509 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:30,509 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:30,525 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:31,015 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:31,016 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:31,031 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:31,032 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:31,033 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:31,045 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:31,051 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:31,052 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:31,074 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:33,754 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:33,755 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:33,772 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:33,775 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:33,777 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:33,796 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:36,669 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:36,670 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:36,682 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:36,684 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:36,684 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:36,697 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:36,703 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:36,703 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:36,717 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:39,987 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:39,987 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:39,999 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:40,004 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:40,004 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:18:40,020 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:18:40,327 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:18:40,327 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:19:14,021 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:19:14,024 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:19:14,026 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:19:14,063 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:19:14,154 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-01-16 21:19:14,157 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Configure stats pid to 5264
2024-01-16 21:19:14,158 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Loading settings from /home/filipeckijan/.config/wandb/settings
2024-01-16 21:19:14,159 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Loading settings from /mnt/c/Users/dziew/Documents/studia/ium/ium-projekt/models/DBSCAN/wandb/settings
2024-01-16 21:19:14,159 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-16 21:19:14,160 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-16 21:19:14,162 INFO    MainThread:5264 [wandb_init.py:_log_setup():526] Logging user logs to /mnt/c/Users/dziew/Documents/studia/ium/ium-projekt/models/DBSCAN/wandb/run-20240116_211914-4ub1fado/logs/debug.log
2024-01-16 21:19:14,164 INFO    MainThread:5264 [wandb_init.py:_log_setup():527] Logging internal logs to /mnt/c/Users/dziew/Documents/studia/ium/ium-projekt/models/DBSCAN/wandb/run-20240116_211914-4ub1fado/logs/debug-internal.log
2024-01-16 21:19:14,165 INFO    MainThread:5264 [wandb_init.py:init():566] calling init triggers
2024-01-16 21:19:14,166 INFO    MainThread:5264 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-01-16 21:19:14,167 INFO    MainThread:5264 [wandb_init.py:init():591] re-initializing run, found existing run on stack: mgatar51
2024-01-16 21:19:14,170 INFO    MainThread:5264 [wandb_run.py:_finish():1962] finishing run ium-team/ium-projekt/mgatar51
2024-01-16 21:19:14,174 INFO    MainThread:5264 [jupyter.py:save_history():445] not saving jupyter history
2024-01-16 21:19:14,175 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:19:14,177 INFO    MainThread:5264 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-01-16 21:19:14,178 INFO    MainThread:5264 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-16 21:19:14,179 INFO    MainThread:5264 [wandb_run.py:_restore():2185] restore
2024-01-16 21:19:14,180 INFO    MainThread:5264 [wandb_run.py:_restore():2191] restore done
2024-01-16 21:19:19,720 INFO    MainThread:5264 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-01-16 21:19:19,721 INFO    MainThread:5264 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-01-16 21:19:19,728 INFO    MainThread:5264 [wandb_run.py:_footer_sync_info():3816] logging synced files
